pkg/providers/common/provider.go

```diff
@@ -32,6 +32,9 @@ func (p *Provider) BindFlags(flags *pflag.FlagSet) {
 	flags.StringVar(
 		&p.BuildVersion, "build-version", "", "Kubernetes Build Version",
 	)
+	flags.StringVar(
```

Comment:
you don't need this, this is to pass variable for the terraform code, any additional variable to the ansible script should be passed via `extra-vars` https://github.com/ppc64le-cloud/kubetest2-plugins/blob/165468f3ac8b3c325ae04526ec1813a861570fe6/kubetest2-tf/deployer/deployer.go#L114

Reply:
oops!! I forgot we have `--extra-vars` flag. Will remove this.